README.md

```diff
@@ -114,19 +114,21 @@ This is a standard procedure: you would have to do it also without using this li
 #### Folder Model
 
 ```typescript
+import * as fs from "fs";
+
 /**
  * Use `const { PKPass } = require("passkit-generator");`
  * for usage in pure Node.js
  */
 import { PKPass } from "passkit-generator";
 
 try {
-	const pass = PKPass.from({
+	const pass = await PKPass.from({
```

Comment:
I know `await` isn't valid outside of an `async` function but I thought it was clearer than the more _correct_ alternatives:

1. Wrap the code sample in an `async` function,
2. Move the subsequent code into `PKPass.from().then(pass => { ... })`

Happy to make either change if you think it is appropriate.

Reply:
I'm okay with `await` like this. Now Node.js has top-level await, isn't it? Also, the previous example was like this.